### PR TITLE
add note and tag class

### PR DESCRIPTION
**Pull request by VityukY**
notes metods:
add
remove
replace
update
find by index
find by sub text
show notes
add tag
remove tag
find note by teg
add full discrp in file

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (VityukY) [reply .vscode/settings.json:1]: done

File: bot_cli/notes_classes.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+- def show_notes(self): - повертає всі теги notes = [{index: {"Note": note, "Tags": tags}}, {index: {"Note": note, "Tags": tags}}] - обовязково звернути увагу на індекси їх юзати для видачі
+- def add_tag(self, index, tag): - додається тег, обовязково треба додати індекс нотатку
+- remove_tag(self, note_index, tag): - вказати у якому нотатку який тег видалити 
+- change_tag(self): - не писав бо редагування тегів не знаю чи треба - 
+"""
+
+from collections import UserDict
+
+
+class Field:
```
[the remark this answers is not in the document]
Comment (VityukY) [reply bot_cli/notes_classes.py:39]: import +

File: bot_cli/notes_classes.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    def __init__(self, value: str):
+        self.value = value
+
+    def __str__(self):
+        return self.value
+
+
+class Note(Field):
+    def __init__(self, value):
+        self.__value = value.capitalize()
```
[the remark this answers is not in the document]
Comment (VityukY) [reply bot_cli/notes_classes.py:49]: import +

File: bot_cli/notes_classes.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        pass  # зробити редагування конретного тега? не впевнений шо треба
+
+    def __str__(self):
+        notes = []
+        for index, note in enumerate(self.data["notes"]):
+            notes.append({index: note["tags"]})
+        return str(notes)
+
+
+notes = Notes()
```
[the remark this answers is not in the document]
Comment (VityukY) [reply bot_cli/notes_classes.py:143]: done

File: bot_cli/notes_classes.py [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    @property
+    def value(self):
+        return self.__value
+
+    @value.setter
+    def value(self, value):
+        self.__value = value.capitalize()
+
+
+class Notes(UserDict):
```
[the remark this answers is not in the document]
Comment (VityukY) [reply bot_cli/notes_classes.py:73]: якшо критично то треба детальніше пояснення

[the remark this answers is not in the document]
Comment (LadaM) [reply bot_cli/notes_classes.py:66]: Not critical at all

**Review by LadaM — APPROVED** [review]

**Pull request merged**